jwst/ami/mask_definition_ami.py

```diff
@@ -0,0 +1,127 @@
+#
+#  Module for defining mask geometry in pupil space
+#  Replace old mask_definitions by reading info in nrm_model into a similar object.
+#
+
+import numpy as np
+import math
+
+from .utils import rotate2dccw
+
+m = 1.0
+mm = 1.0e-3 * m
+um = 1.0e-6 * m
+
+
+class NRMDefinition():
+
+	def __init__(self, nrm_model, maskname='jwst_ami', rotdeg=None, chooseholes=None):
+		"""
+		Set attributes of NRMDefinition class. 
+
+		Get hole centers and other mask geometry details from NRMModel, apply rotations/flips 
+		as necessary and set them as attributes.
+
+		Parameters
+		----------
+		nrm_model: NRMModel
+			datamodel containing NRM reference file data
+		maskname: string
+			Identifier for mask geometry; default 'jwst_ami', optional
+		rotdeg: list of floats
+			range of rotations to search (degrees), optional
+		chooseholes: list
+			None, or e.g. ['B2', 'B4', 'B5', 'B6'] for a four-hole mask, optional
+			If None, use real seven-hole mask
+		"""
+
+		if maskname not in ['jwst_ami','jwst_g7s6c']:
```

Comment:
It looks like you removed `jwst_g7s6c` elsewhere.  Just checking here if it's still a valid input... are the two masks identical?

Reply:
I kept it since the technical name of the mask design is jwst_g7s6c, and the standalone version of the code uses that so it was kind of an attempt at compatibility with that, but they do refer to the same thing.